### PR TITLE
d3 misc edits 

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Arcueid/Day 3/03_00_ARC03_5_1.txt [hunk 1]
```diff
@@ -258,7 +258,7 @@ I've heard that it takes an entire day of hard work to dismember a single human
 [sha:071ca558aaa05201a41ba3de09f06fe2a65b9689]{
 -- Page 262, Offset 4709.
 -- だから、こんなナイフ一本で出来る筈がない。
-So there's no way you could do it with just a knife like this.
+So there's no way you could do it as simply as this; armed with nothing but a knife.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 3/03_00_ARC03_5_1.txt:261]: Not sure there's enough here to warrant two full clauses
> There's no way you could manage it with nothing more than a knife.

File: script/Arcueid/Day 3/03_00_ARC03_1.txt [hunk 2]
```diff
@@ -132,6 +132,7 @@ I awake from my reverie.
 [sha:f6eb0bcf927f1c91306b911326c9fa5a3c5c19a2]{
 -- Page 8, Offset 3969.
 -- 目蓋ごしに朝の光を感じ取って、目を閉じたまま枕元に手を這わせる。
+// we have 枕元 as bedside table in day 2, so probably best to agree upon a consistent translation for it
```
Comment (rschlaikjer) [inline script/Arcueid/Day 3/03_00_ARC03_1.txt:135]: Either fine with me, nightstand seems more ngrammed

Comment (nolo16) [reply script/Arcueid/Day 3/03_00_ARC03_1.txt:135]: i'll go ahead and change the other 枕元 instances to nightstand then

Comment (nolo16) [reply script/Arcueid/Day 3/03_00_ARC03_1.txt:135]: we also have variations of "near my pillow" in other files, do we know if there's like a genuine desk/stand he puts his glasses on?

Comment (rschlaikjer) [reply script/Arcueid/Day 3/03_00_ARC03_1.txt:135]: I _think_ I remember looking this up and seeing one in the CGs

Comment (nolo16) [reply script/Arcueid/Day 3/03_00_ARC03_1.txt:135]: image

yeah, it's there

nightstand it is then

File: script/Arcueid/Day 3/03_00_ARC03_1_2.txt [hunk 3]
```diff
@@ -135,12 +135,13 @@ Ah, since you get along so well, would you like to be her valet? I would not min
 [sha:1c79225e0fa222e8320b30c53d9ea6dbca35957f]{
 -- Page 53, Offset 4080.
 -- この妹の<棘|パワー>がすごい。
-My sister's <power|b i t i n g  w o r d s     > is truly formidable.
+//don't love this but don't love the ruby either
+My sister's power――her scathing remarks――is truly formidable.
```
Comment (Avuxo) [inline script/Arcueid/Day 3/03_00_ARC03_1_2.txt:139]: this is pretty stilted. If you're so inclined you can just turnn "power" into "scathing remarks" or whatever. just reflow the sentence if you do

**Review by Avuxo — APPROVED** [review]

**Review by Volvo-hackangel — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 4]
```diff
@@ -31,7 +31,7 @@ As I bask in the morning's warm welcome―――
 [sha:6cbbd7b766f75ada376a8dfb0bfea9c54ddc9de8]{
 -- Page 2, Offset 24265.
 -- 無意識のうちに枕元の眼鏡を手に取る。
-I put on the glasses resting by my pillow,
+I put on the glasses resting by my nightstand,
```
Comment (yk1512) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:34]: on my nightstand

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 3/03_00_ARC03_1.txt (hunk 2 above)
Comment (rschlaikjer) [inline script/Arcueid/Day 3/03_00_ARC03_1.txt:135]: Comment can probable go now

**Review by rschlaikjer — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_1.txt (hunk 4 above)
Comment (nolo16) [reply script/Arcueid/Day 10/10_00_ARC10_1.txt:34]: completely missed this comment yesterday sorry :aichanteehee:

**Pull request merged**